packages/core/components/divider/index.js

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import { color, space, themeGet } from "styled-system";
+import { Direction } from "../../common/direction";
+
+const grayLight1 = "#C3CADF";
+
+const HDivider = styled.hr`
+  ${space}
+  ${color}
+  background-color: ${props => themeGet(`colors.${props.color}`, grayLight1)};
```

Comment:
So I think that we may want to either provide a default color prop ("grayLight1"), or fallback to a theme property here instead of a hardcoded color. So basically, 
```suggestion
  background-color: ${props => themeGet(`colors.${props.color}`, themeGet('colors.grayLight1')};
```

Reply:
Ah yes. Totally.